src/Illuminate/Routing/RoutingServiceProvider.php

```diff
@@ -154,9 +158,12 @@ protected function registerPsrResponse()
             if (class_exists(NyholmPsrResponse::class)) {
                 return new NyholmPsrResponse;
             }
+
             if (class_exists(ZendPsrResponse::class)) {
                 return new ZendPsrResponse;
             }
+
+            throw new Exception('Unable to resolve PSR response. Please install nyholm/psr7 or laminas/laminas-diactoros.');
```

Comment:
Since we're not actually supporting `zendframework/zend-diactoros` as per the previous discussion, and the zend stuff is now deprecated, maybe we should only recommend `nyholm/psr7` here?

Reply:
So, we'll have:

```
Unable to resolve PSR response. Please install nyholm/psr7.
```